viz_scripts/scaffolding.py

```diff
@@ -207,3 +297,88 @@ def CO2_impact_lb(df,distance,col1,col2):
     df['CO2_Impact(lb)']  = round((df[col1+'_lb_CO2'] - df[col2+'_lb_CO2']),3)
   
     return df
+
+
+def cost_impact(data, dist, repm, mode):
+    """
+    Calculates the cost impact for participants in OpenPATH
+    
+     Parameters:
+        data - participant trip data from OpenPATH
+        dist - feature name in df of feature with distance in miles
+        repm - feature name in df of feature with replaced mode
+        mode - feature name in df of feature with confirmed mode
+        
+    Returns:
+        data with appended cost impact feature for each trip in $$$ (float)
+    """
+  
+    data[mode+'_cost'] = data[dist] * data['cost__trip_'+mode]
+    data[repm+'_cost'] = data[dist] * data['cost__trip_'+repm]
+    data['Cost_Impact($)'] = round((data[mode+'_cost'] - data[repm+'_cost']),2)
```

Comment:
This is indeed as simple as flipping the values. Note for e-bikes, mode_cost is always zero, and repm_cost is always > 0, so the value will always be negative.

If you look carefully at the existing energy impact calculation, 
```
df['CO2_Impact(lb)']  = round((df[col1+'_lb_CO2'] - df[col2+'_lb_CO2']),3)
```

note that col1 is the replaced mode

```
     conditions_col1 = [(df['Replaced_mode_fuel'] =='gasoline'),
```